app/models/agents/webhook_agent.rb

```diff
@@ -88,7 +89,11 @@ def receive_web_request(params, method, format)
         create_event(payload: payload)
       end
 
-      [interpolated(params)['response'] || 'Event Created', code]
+      if interpolated['response_headers'].presence
```

Comment:
Is this conditional needed? Since you check for the header presence in the `WebRequestsController` is think you can return just `interpolated['response_headers'].presence` in the array.